### PR TITLE
[FLINK-33650][doc] Add the Checkpoint Storage part for Chinese Checkpoint page

### PR DESCRIPTION
**Pull request by 1996fanrui**
- [FLINK-33650][doc] Remove manage memory part in FileSystemCheckpointStorage due to it can be used for rocksdb state backend.
- [FLINK-33650][doc] Add the Checkpoint Storage part for Chinese Checkpoint page

### CODE REVIEW

File: docs/content/docs/ops/state/checkpoints.md [hunk 1]
```diff
@@ -86,8 +86,6 @@ The `FileSystemCheckpointStorage` is encouraged for:
 
   - All high-availability setups.
 
-It is also recommended to set [managed memory]({{< ref "docs/deployment/memory/mem_setup_tm" >}}#managed-memory) to zero.
```
Comment (JingGe) [inline docs/content/docs/ops/state/checkpoints.md:89]: may I know why this is removed?

**Review by JingGe — COMMENTED** [review]
Thanks @1996fanrui for adding the content.

**Review by JingGe — APPROVED** [review]
LGTM

**Comment by 1996fanrui** [comment]
Thanks @JingGe  for the review, merging~

**Pull request merged**